frontend/sass/_side-nav.scss

```diff
@@ -13,7 +13,20 @@
     &:hover svg g {
       fill: $link-color-active;
     }
+
+    &.side-nav-active-item {
+      background-color: #2d476a;
+      color: white !important;
+      padding: 5px !important;
```

Comment:
Is there a way this can be done without using `!important` ?

Reply:
Probably, it will just involve a bunch of specificity selectors to override USWDS behavior